packages/grid_client/src/modules/networks.ts

```diff
@@ -26,7 +26,9 @@ class NetworkModule extends BaseModule {
       options.ipRange,
       options.nodeId,
       options.solutionProviderId!,
+      options.mycelium,
       options.description,
+      options.myceliumSeed!,
```

Comment:
this one field is `string` while this method is waiting for `MyceliumNetworkModel[]`